<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerView</class>
 <widget class="QMainWindow" name="ViewerView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewerView</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-color: rgb(29, 38, 51)}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="bg_settings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 12pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Background settings:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(68, 235, 153);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="bg_colour">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}
</string>
    </property>
    <property name="text">
     <string>colour:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="background_colour">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="line_settings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 12pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Line settings:</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(68, 235, 153);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="line_colour_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>colour:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="line_colour">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>style:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="line_style">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vertex_settings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 12pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Vertex settings:</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="vertex_style">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="vertex_colour">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>480</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="vertex_colour_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>colour:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vertex_size_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vertex_style_text">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}v</string>
    </property>
    <property name="text">
     <string>style:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="line_spinbox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="vertex_spinbox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>600</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>30</y>
      <width>211</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>70</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse file</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>580</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {color: rgb(68, 235, 153); 
font: 700 9pt &quot;Arial&quot; ;}
</string>
    </property>
    <property name="text">
     <string>Parallel projection</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>610</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Central projection</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>120</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="line_colour_text_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>X axis:</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_settings_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>100</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 12pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Translate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>200</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Y axis:</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>260</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Z axis:</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>170</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>230</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>300</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>140</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>200</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>260</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>460</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>400</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="line_colour_text_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>X axis:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>490</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>430</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="line_settings_3">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>330</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 12pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Rotate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>822</x>
      <y>370</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_6">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>530</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>350</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text_6">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>490</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Z axis:</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text_7">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>430</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Y axis:</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>580</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="line_settings_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>560</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 12pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Scale:</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_7">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>610</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>580</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>File name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>600</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Vertices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="line_style_text_10">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>620</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>Edges:</string>
    </property>
   </widget>
   <widget class="QLabel" name="filename_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>580</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial white</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="vertices_num">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial white</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="edges_num">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>620</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial white</family>
      <pointsize>9</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {color: rgb(68, 235, 153);
font: 700 9pt &quot;Arial&quot; white;}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>511</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>962</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
